lib/pf/Switch/Ubiquiti/Unifi.pm

```diff
@@ -41,6 +46,10 @@ sub description { 'Unifi Controller' }
 # CAPABILITIES
 # access technology supported
 sub supportsExternalPortal { return $TRUE; }
+sub supportsWirelessDot1x { return $TRUE; }
+sub supportsWirelessMacAuth { return $TRUE; }
+# inline capabilities
+sub inlineCapabilities { return ($MAC,$SSID); }
```

Comment:
really need that ?

Reply:
Yes, btw i only know one guy who use that.